compatimetrics.py

```diff
@@ -0,0 +1,501 @@
+    """
+    if positive is None:
+        positive = y.squeeze().unique()[1]
+    T = 0
+    T_disagreement = 0
+    for i in range(len(y)):
+        if y[i] == positive:
+            T += 1
+            if pred1[i] != pred2[i]:
+                T_disagreement += 1
+    return T_disagreement/T
+
+def conjunctive_accuracy(pred1, pred2, y):
+    """ Calculates conjunctive accuracy of two prediction vectors
+    and true values of variable.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive accuracy value of given predictions pair
+        and true values vector
+    """
+    same = 0
+    for i in range(len(y)):
+        if pred1[i] == pred2[i] and pred2[i] == y[i]:
+            same += 1
+    return same/len(y)
+
+def conjunctive_precission(pred1, pred2, y, positive=None):
+    """ Calculates conjunctive precision of two prediction vectors
+    and true values of variable.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+    positive: value, optional
+        value that represents a positive observation in variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive precision value of given predictions pair
+        and true values vector
+    """
+    if positive is None:
+        positive = y.squeeze().unique()[1]
+    TTP = 0
+    FFN = 0
+    for i in range(len(y)):
+        if pred1[i] == positive and pred2[i] == positive:
+            if y[i] == positive:
+                TTP += 1
+            else:
+                FFN += 1
+    if TTP+FFN == 0:
+        return 0
+    return TTP/(TTP+FFN)
+
+def conjunctive_recall(pred1, pred2, y, positive=None):
+    """ Calculates conjunctive recall of two prediction vectors
+    and true values of variable.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+    positive: value, optional
+        value that represents a positive observation in variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive recall value of given predictions pair
+        and true values vector
+    """
+    if positive is None:
+        positive = y.squeeze().unique()[1]
```

Comment:
Shouldn't there be is not None, and some condition what to do when there is none?